.github/workflows/ci.yml

```
@@ -145,15 +145,67 @@ jobs:
+      fail-fast: false
+      matrix:
+        node: ["22", "20", "18"]
+        os: [ubuntu-latest, windows-latest]
+        vite: ["5", "6", "7"]
+        exclude:
+          - os: windows-latest
+            node: 20
+          - os: windows-latest
+            node: 18
+          - vite: 7
+            node: 19
```

actionlint (1.7.7)

value "19" in "exclude" does not match in matrix "node" combinations. possible values are "22", "20", "18"

(matrix)